test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -165,21 +165,19 @@ var _ = Describe("Azure Container Cluster using the Kubernetes Orchestrator", fu
 			if cfg.BlockSSHPort {
 				Skip("SSH port is blocked")
 			} else {
-				var nodeList *node.List
+				var nodes []node.Node
 				var err error
 				if !eng.ExpandedDefinition.Properties.HasLowPriorityScaleset() {
-					nodeList, err = node.GetReady()
+					nodes, err = node.GetWithRetry(1*time.Second, cfg.Timeout)
```

Comment:
These changes came about due to poking at cilium clusters w/ E2E and seeing flakes due to the fact that a cilium cluster configuration initiates an OS reboot (thus the nodes may go offline during these tests and we need to retry).